pkg/helm/actions/get_release.go

```diff
@@ -0,0 +1,16 @@
+package actions
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+func GetRelease(releaseName string, conf *action.Configuration) (*release.Release, error) {
```

Comment:
Does this method need to be public?

Reply:
Yes, Since it will used by `handler` package.